### PR TITLE
add POPM frame with test

### PR DESCRIPTION
**Pull request by m13m**

### CODE REVIEW

**Comment by m13m** [comment]
stole some of the counter stuff from PCNT frame.

File: test/id3vx/encoding_test.exs [hunk 1]
```diff
@@ -533,4 +533,25 @@ defmodule Id3vx.EncodingTest do
...
+    }
+
+    binary = Frame.encode_frame(frame, %Tag{version: 3})
+    assert <<frame_header::binary-size(10), frame_data::binary>> = binary
+    assert <<"POPM", frame_size::size(32), _flags::binary-size(2)>> = frame_header
+    assert 25 == frame_size
+
+    [email, rest] = :binary.split(frame_data, <<0, 0>>)
+
+    assert email == frame.data.email
```
Comment (lawik) [inline test/id3vx/encoding_test.exs:555]: not checking rating or count

**Comment by lawik** [comment]
Is it done? Please provide a comment if requesting review.

**Comment by m13m** [comment]
yes @lawik this is also ready for review.

**Comment by m13m** [comment]
This PR is review ready ✨ 

- added test with id3v2 cli for existence of POPM frame 

- ffmpeg doesn't support particular frame.

File: lib/id3vx/frame.ex [hunk 2]
```diff
@@ -674,6 +674,26 @@ defmodule Id3vx.Frame do
...
+      counter: counter
+    } = frame.data
+
+    # byte size of counter
+    counter_binary_size = counter |> :binary.encode_unsigned() |> byte_size()
+
+    # use minimum 32 bits
+    counter_size = max(counter_binary_size * 8, 32)
+
+    frame_binary = [email, <<0, 0>>, <<rating::size(8)>>, <<counter::size(counter_size)>>]
```
Comment (lawik) [inline lib/id3vx/frame.ex:690]: Should be single null byte, right?

File: lib/id3vx/frame.ex [hunk 3]
```diff
@@ -1044,9 +1064,24 @@ defmodule Id3vx.Frame do
     }
   end
 
-  def parse(id, _tag, _flags, _data) do
-    Logger.warn("Unimplemented frame parsed: #{id}")
+  def parse("POPM" = id, _tag, _flags, data) do
+    [email, rest] = :binary.split(data, <<0, 0>>)
```
Comment (lawik) [inline lib/id3vx/frame.ex:1068]: Single null byte :)

**Review by lawik — CHANGES_REQUESTED** [review]

**Comment by m13m** [comment]
FIXED the null byte issue and resolved the conflict

as it had no other issue apart from that I merging this PR.

**Pull request merged**